x/concentrated-liquidity/pool_test.go

```diff
@@ -1141,3 +165,15 @@ func (s *KeeperTestSuite) TestConvertConcentratedToPoolInterface() {
 	_, err := cl.ConvertConcentratedToPoolInterface(concentratedPool)
 	s.Require().NoError(err)
 }
+
+func (s *KeeperTestSuite) TestPoolIToConcentratedPool() {
```

Comment:
I see that it is. Can we also add a test that if we call this function with a stableswap pool, it fails